Slam/ShowQueueViewController.swift

```diff
@@ -20,10 +20,14 @@ public class ShowQueueViewController: UIViewController {
         }
     }
 
-    public func initializeDelegates() {
-        queue = QueueView(display: currentQueue)
-        queueManager = APIManagedQueue(queueView: queue!)
-        currentQueue!.dataSource = queue
+    public func initializeDelegates(manager:QueueManager = APIManagedQueue()) {
+        queueManager = manager
+        queueView = QueueView(display: currentQueue)
+        queueManager!.setQueueView(queueView!)
```

Comment:
This could be `queueManager!.queueView = queueView!` to remove the need for `setQueueView`.